modules/apps/29-fee/keeper/relay.go

```diff
@@ -19,11 +20,17 @@ func (k Keeper) SendPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability,
 func (k Keeper) WriteAcknowledgement(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet ibcexported.PacketI, acknowledgement []byte) error {
 	// retrieve the forward relayer that was stored in `onRecvPacket`
 	packetId := channeltypes.NewPacketId(packet.GetSourceChannel(), packet.GetSourcePort(), packet.GetSequence())
+
+	// relayer address returned here is the
 	relayer, _ := k.GetForwardRelayerAddress(ctx, packetId)
```

Comment:
@AdityaSripal Can you remind me again why you don't want to error here? I'm throwing an error below if not found, wondering if you want to omit this also?